src/utils/bioschemasMapper.js

```diff
@@ -0,0 +1,196 @@
+import { getBaseUrl } from '@/utils/'
+
+// Should be better to put in the data model
```

Comment:
Can this be done for this PR?

Reply:
We should ask the state of this to the data team.

Reply:
At the moment this is only added for materials and data_categories. You can retrieve the data with a GET request to https://directory.bbmri-eric.eu/api/v2/eu_bbmri_eric_material_types and  https://directory.bbmri-eric.eu/api/v2/eu_bbmri_eric_data_types respectively.
I have created a story for our team to add the IRIs also for sex (you would get the values from  https://directory.bbmri-eric.eu/api/v2/eu_bbmri_eric_sex_types) and to add tags to the properties.

I would suggest that for now this mapping is okay, but we amend it when this is done.